src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/ScopeTab.tsx

```diff
@@ -14,7 +14,7 @@ const ScopeTab = ({ workPackage }: { workPackage: WorkPackage }) => {
         items={workPackage.expectedActivities
           .filter((ea) => !ea.dateDeleted)
           .map((ea): CheckListItem => {
-            return { ...ea, resolved: !!ea.userChecked, user: ea.userChecked, dateAdded: ea.dateAdded };
+            return { ...ea, resolved: !!ea.userChecked, user: ea.userChecked, dateChecked: ea.dateChecked };
```

Comment:
u should be able to just delete dateChecked

Reply:
Oh I'm so dumb, sean said the same thing, y'all meant bc of the spread operator, I'll make the change